src/Compiler/FSComp.txt

```diff
@@ -1789,5 +1789,6 @@ featureUseTypeSubsumptionCache,"Use type conversion cache during compilation"
 3872,tcPartialActivePattern,"Multi-case partial active patterns are not supported. Consider using a single-case partial active pattern or a full active pattern."
 featureDontWarnOnUppercaseIdentifiersInBindingPatterns,"Don't warn on uppercase identifiers in binding patterns"
 3873,chkDeprecatePlacesWhereSeqCanBeOmitted,"This construct is deprecated. Sequence expressions should be of the form 'seq {{ ... }}'"
+3874,tcExpectedTypeParamMarkedWithUnitOfMeasureAttribute,"Expected unit-of-measure type parameter must be marked with the [<Measure>] attribute."
```

Comment:
Im not entirely convinced with this wording. Any suggestion @psfinaki ?

Reply:
I'd say - as long as this error pops up only in the certain scenario that the PR addresses, this is a reasonable wording.

Reply:
Yeah this will only be shown when expecting a measure type parameter in a type augmentation. See tests